tools/gfx/immediate-renderer-base.cpp

```diff
@@ -57,8 +57,8 @@ class CommandBufferImpl : public ICommandBuffer, public RefObject
             *outObject = nullptr;
             return SLANG_E_NO_INTERFACE;
         }
-        virtual SLANG_NO_THROW uint32_t SLANG_MCALL addRef() { return 1; }
-        virtual SLANG_NO_THROW uint32_t SLANG_MCALL release() { return 1; }
+        virtual SLANG_NO_THROW uint32_t SLANG_MCALL addRef() override { return 1; }
+        virtual SLANG_NO_THROW uint32_t SLANG_MCALL release() override { return 1; }
```

Comment:
Is there a reason for using `override` over `SLANG_OVERRIDE`. I guess we might be at the point where all the compilers we can work with support `override` so that there is no point in trying to be defensive against older compilers...

Reply:
Yeah, I doubt if there are any compilers that doesn't support override.